### PR TITLE
OPHJOD-296: Add toast component

### PR DESCRIPTION
**Pull request by sauanto**
## Description

- Add toast component

## Related JIRA ticket

https://jira.eduuni.fi/browse/OPHJOD-296

### CODE REVIEW

File: lib/components/Toast/Toast.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+export interface ToastProps {
+  /** Text shown on the toast */
+  text: string;
+  /** Icon shown on the toast */
+  icon?: string;
+  /** Aria label for the icon */
+  iconAriaLabel?: string;
+  variant?: 'success' | 'warning' | 'error';
```
Comment (ketsappi) [inline lib/components/Toast/Toast.tsx:8]: Maybe would be good to tell somewhere in the Storybook if not in JSDoc, that color is not enough to communicate the "priority" vs other variants so icon should be a must.

File: lib/components/Toast/Toast.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+export interface ToastProps {
```
Comment (ketsappi) [inline lib/components/Toast/Toast.tsx:1]: Would it make sense in Toast that we lock down the color and icon combinations? 
Success = check_circle, etc?

**Review by ketsappi — APPROVED** [review]

File: lib/components/Toast/Toast.tsx (hunk 1 above)
Comment (sauanto) [reply lib/components/Toast/Toast.tsx]: Fixed now

File: lib/components/Toast/Toast.tsx (hunk 2 above)
Comment (sauanto) [reply lib/components/Toast/Toast.tsx:1]: Would probably make sense. Need more information from the designer on that.

**Pull request merged**